### ISSUE TITLE
Tiny refactorings from #5167

### ISSUE THREAD

**Issue by chillenzer** [ComputationalRadiationPhysics/picongpu#5169]
There are some tiny refactorings like re-arranging functions, removing clang-format comments, etc. that were not addressed in #5167 due to a different focus there. Someone should come back to those.

**Comment by Kushagra1480** [comment]
Hi! Is this issue open to being worked on? I'd love to work on it!

**Comment by chillenzer** [comment]
Yes, this is still open and nobody's working on it. Thanks for your initiative!

**Comment by Kushagra1480** [comment]
Hi @chillenzer! A couple of questions:
- Should I remove all the clang-format comments? If not, which ones should I keep?
- What does re-arranging the functions mean here?

Could you help me by listing out all the things that I need to do in this issue? Thanks for your help!

**Comment by chillenzer** [comment]
Well, after having a closer look at it the refactorings are really tiny:
- Removing the two explicitly mentioned `clang-format` comments. (If you find more that are unnecessary, feel free to remove them as well.)
- Might be an option to move `closeSeries` wherever `openSeries` went but it's not really necessary.
